config/sync/views.view.landing_page.yml

```diff
@@ -169,26 +169,9 @@ display:
           entity_type: node
           entity_field: langcode
       pager:
-        type: full
+        type: none
```

Comment:
Hmm, is this view used anywhere at all? As I can see this view was the original front page but if I'm not mistaken you no longer use this and in that case this can be removed.

Reply:
Deleted.